File: Beecrowd/1013 O Maior/main.cpp
#include <iostream>

using namespace std;

int main() {

    int a, b, c, MaiorAB;

    cin >> a;
    cin >> b;
    cin >> c;

    MaiorAB = (a + b + abs(a - b)) / 2;
    MaiorAB = (MaiorAB + c + abs(MaiorAB - c)) / 2;

    cout << MaiorAB << " eh o maior" << endl;

    return 0;
}

File: Beecrowd/1013 O Maior/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int a, b, c, MaiorAB;

        a = sc.nextInt();
        b = sc.nextInt();
        c = sc.nextInt();

        MaiorAB = (a + b + Math.abs(a - b)) / 2;
        MaiorAB = (MaiorAB + c + Math.abs(MaiorAB - c)) / 2;

        System.out.println(MaiorAB + " eh o maior");

        sc.close();
    }

}